src/platform/site-wide/sass/merger.scss

```diff
@@ -739,7 +775,43 @@
   .white {
     color: $color-white;
   }
+
+  #va-detailpage-sidebar {
+    color: $color-link-default;
+
+    .usa-accordion-button {
+      color: $color-link-default;
+      font-weight: normal;
+      font-size: 17px;
+
+      &:hover, &:focus {
+        background-color: $mm-gray;
+        font-weight: bold;
+      }
+
+      &[aria-expanded=false] {
+        background-image: url("/img/plus-blue.svg");
+      }
+
+      &[aria-expanded=true] {
+        background-image: url("/img/minus-blue.svg");
+      }
+    }
+  }
+
+  .sidenav-previous-page {
+    padding-bottom: 15px;
+
+    a {
+      &::before {
+        content: " \2039 ";
+        display: inline-block;
+        padding: 0 0.35em 0 0;
+      }
+    }
+  }
 }
+
```

Comment:
Ah man, do we still not have a dedicated sass file for the side bar? Hate to see this file keep growing...

Reply:
If you consider it to be risky, don't worry about it at all. We can wait til after launch.